### PR TITLE
file.coffee, nsfw.rest & unscan

### PR DESCRIPTION
**Pull request by abcdan**
* Added file.coffee where developers can store their files
* Added nsfw.rest for NSFW scanning and malware detection
* Added unscan for NSFW scanning and malware detection

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -264,6 +265,7 @@ Table of Contents
   * [microenv.com](https://microenv.com) —  Create fake REST API for developers with possibility to generate code and app in docker container.
   * [neptune.ai](https://neptune.ai/) - Log, store, display, organize, compare and query all your MLOps metadata. Free for individuals: 1 member, 100 GB of metadata storage, 200h of monitoring/month
   * [News API](https://newsapi.org) — Search news on the web with code, get JSON results. Developers get 3,000 queries free each month.
+  * [nsfw.rest](https://nsfw.rest) - Free API to scan images for NSFW content and files for malware. 100 request per second.
```
Comment (ripienaar) [inline README.md:268]: there doesnt seem much point to this, the unscan one is by the same people and does more, is this one just there for SEO or something?

**Comment by abcdan** [comment]
They are essentially the same, nsfw.rest can be removed :)



On May 15, 2022, 5:38 PM, at 5:38 PM, "R.I.Pienaar" ***@***.***> wrote:
***@***.*** commented on this pull request.
>
>
>
>> @@ -264,6 +265,7 @@ Table of Contents
>* [microenv.com](https://microenv.com) —  Create fake REST API for
>developers with possibility to generate code and app in docker
>container.
>* [neptune.ai](https://neptune.ai/) - Log, store, display, organize,
>compare and query all your MLOps metadata. Free for individuals: 1
>member, 100 GB of metadata storage, 200h of monitoring/month
>* [News API](https://newsapi.org) — Search news on the web with code,
>get JSON results. Developers get 3,000 queries free each month.
>+  * [nsfw.rest](https://nsfw.rest) - Free API to scan images for NSFW
>content and files for malware. 100 request per second.
>
>there doesnt seem much point to this, the unscan one is by the same
>people and does more, is this one just there for SEO or something?
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/ripienaar/free-for-dev/pull/2445#pullrequestreview-973189070
>You are receiving this because you authored the thread.
>
>Message ID:
***@***.***>

**Comment by ripienaar** [comment]
thank you

**Comment by abcdan** [comment]
> thank you

No problem!

**Comment by imba-tjd** [comment]
How about moving file.coffee to `storage-and-media-processing`?

**Pull request merged**